docs/apps/Broker.md

```diff
@@ -18,13 +18,13 @@ command line only:
 
 configuration:
   -n [ --name ] arg      name of the broker
-  -t [ --coretype ] arg      type of the broker ("(zmq)", "ipc", "test", "mpi","test", "tcp", "udp")
+  -t [ --type ] arg      type of the broker ("(zmq)", "ipc", "test", "mpi","test", "tcp", "udp")
```

Comment:
It looks like the command line arguments in the documentation for the helics apps have been reverted to use the (as of v3) old names.

Reply:
did you make those changes in the change name branch or the v3userguide branch?

Reply:
I didn't intend to make those changes outside the trim_config_options branch. The line that is being changed, though, regardless of how it got there, is the (new) correct name and the edit is reverting it to the old name.

Reply:
Basically the files got moved so the merge didn't capture the updates yet.  I will work on that.

Reply:
Maybe the change was made to helics3 when merging in the trim_config_options and then v3userguide was updated with those changes?